config/default/pre-commit-config.yaml.j2

```
@@ -71,6 +71,16 @@ repos:
     rev: "6.0.0"
     hooks:
     -   id: i18ndude
+        exclude: "%(i18ndude_extra_lines)s"
```

```suggestion
%(i18ndude_extra_lines)s
```

Sorry I missed that, we need to change this, otherwise we would only be able to exclude files from being analyzed by `i18ndude` while some might want to install other dependencies of `i18ndude` or apply some other configuration options to it 👍🏾